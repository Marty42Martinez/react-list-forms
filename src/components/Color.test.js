import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color Component', () => {
  it('renders a Color', () => {
    const rgb = {
      red: 0,
      green: 200,
      blue: 120
    };
    const wrapper = shallow(<Color name="Green-ish" rgb={rgb} />);
    expect(wrapper).toMatchSnapshot();
  });

});
