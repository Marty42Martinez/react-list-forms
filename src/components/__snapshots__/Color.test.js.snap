// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Component renders a Color 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Green-ish
     
    <div
      style={
        Object {
          "backgroundColor": "#00C878",
          "display": "inline-block",
          "height": "0.8em",
          "width": "0.8em",
        }
      }
    />
  </dd>
  <dt>
    Hex
  </dt>
  <dd>
    #00C878
  </dd>
  <dt>
    RGB
  </dt>
  <dd>
    <p>
      Red - 
      0
    </p>
    <p>
      Green - 
      200
    </p>
    <p>
      Blue - 
      120
    </p>
  </dd>
</dl>
`;
